packages/icons/web/v2/forward_24.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="25" height="24" viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The SVG's `width`, `height` and `viewBox` attributes must match the icon's size exactly. This SVG is 25px (instead of 24px) wide. This is probably caused by the frame being placed on a half-pixel in Figma.